### ISSUE TITLE
Print default port

### ISSUE THREAD

**Issue by 89z** [sergiotapia/torrentinim#11]
If I start the program, I just get this:

~~~
[eztv] Starting EZTV crawl
Doing it
[1337x] Starting 1337x crawl
[1337x] Download html: https://1337x.to/cat/Anime/1/
[nyaa] Starting Nyaa crawl
[nyaa-pantsu] Starting NyaaPantsu crawl
[nyaa-sukebei] Starting Nyaa Sukebei crawl
[yts] Starting YTS crawl
[torrentdownloads] Starting TorrentDownloads.me crawl
[thepiratebay] Starting ThePirateBay crawl
[rarbg] Starting Rarbg crawl
[1337x] Download html: https://1337x.to/torrent/4880927/HR-Shakunetsu-Kabaddi-S01E09-Web-1080p-HEVC-OPUS-HR-RG/
[nyaa-pantsu] Crawler error, restarting...
[nyaa-pantsu] Starting NyaaPantsu crawl
~~~

So its not clear where to even browse to see the results

**Comment by 89z** [comment]
Looks like the current default is here:

https://github.com/sergiotapia/torrentinim/blob/929f09c9ddbc504502a88755760a54175ca4cce0/src/torrentinim.nim#L35

**Comment by sergiotapia** [comment]
Thank you for reporting this, it's now fixed in v0.3.1! 

https://github.com/sergiotapia/torrentinim/releases/tag/v0.3.1
